### PR TITLE
Rs 2022.10.31

### PR DESCRIPTION
**Pull request by rschlaikjer**
Spooky ~sexy~ scary skele-nuns send shivers down your spine

2022-10-31-221514_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 9/09_01_CIEL09_3_1.txt [hunk 1]
```diff
@@ -207,27 +223,29 @@ I hear the woman breathing behind me.
...
 -- Page 337, Offset 23006.
 -- というより、特に何の意見も無い。
-To be more precise, I'm pretty indifferent about it.
+To be more precise, I don't really care how she does it at all.
 }
 [sha:73e864c5920953f64ec334ca06ae896993148977]{
 -- Page 338, Offset 23007.
 -- 「後はコレを仕舞えばいいんですよね。
-"All that's left to do is store it away, right?
+"All that's left is to pack this guy up, right?
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_3_1.txt:248]: At this late stage we can't be callin' him "this guy"!
>...pack this thing up, right?"

**Review by Petrikow — COMMENTED** [review]
otu

File: script/Ciel/Day 9/09_01_CIEL09_3_1.txt [hunk 2]
```diff
@@ -272,22 +292,24 @@ Was my performance that impressive?
...
 -- 「――――――、はあ―――」
-"Hah―――――"
+"%{i}Sigh%{/i}―――"
 }
 [sha:5c3bc86a794837012b81ef155ebf46131fc11d4b]{
 -- Page 342, Offset 23019.
 -- しかし、なるほど。
-Still, so that's how it is.
+// Bit unsure how best to null platitude this into EN.
+// Assuming it's him reflecting on the action ('so that happened').
```
Comment (Petrikow) [inline script/Ciel/Day 9/09_01_CIEL09_3_1.txt:311]: Yeah, something like a realization of how he quite liked it.

File: script/Ciel/Day 9/09_01_CIEL09_3_1.txt [hunk 3]
```diff
@@ -1,7 +1,9 @@
 [sha:5c739aa6b10dc78abb740cc9ad4c4fd7d6ee3733]{
 -- Page 10009, Offset 22962.
 -- 「なに？　やっぱり、シエルさんと学生ごっこやってる志貴クンには荷が重い？」
-"What? Too much to bear for you? Still playing make-believe student like Ciel-san?"
+"What?
+ Let me guess,
+ you don't want to dirty your hands so you can keep playing at being a student with Ciel-san?"
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 9/09_01_CIEL09_3_1.txt:6]: Yeah, I felt the need to modify 荷が重い a bit for it to read nicely. Straight up 'playing ~house~ school' also doesn't connect as well as I'd like. Either of these better for you?
> Or what, would doing that shatter the illusion you have of a happy school life with Ciel-san?
> Or what, are you too engrossed in your fantasy of living a normal school life with Ciel-san that you couldn't even consider it?

[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 9/09_01_CIEL09_3_1.txt:6]: It feels more like a kind of insult, specifically toward Shiki. It's not so much about shattering his illusion, or whatever, but more just a way for her to insinuate that he's a naive baby.

To go with a more liberal example:
>Or what? Too much for Ciel's little playmate? Too busy playing make-believe with her at school?

**Pull request merged**